soilmask.py

```diff
@@ -42,7 +43,8 @@ def prepare_metadata_for_geotiff(transformer_info: dict = None) -> dict:
             extra_metadata["transformer_version"] = str(transformer_info.get("version", ""))
             extra_metadata["transformer_author"] = str(transformer_info.get("author", ""))
             extra_metadata["transformer_description"] = str(transformer_info.get("description", ""))
-            if "repository" in transformer_info and "repUrl" in transformer_info["repository"]:
+            if "repository" in transformer_info and transformer_info["repository"] and \
```

Comment:
```suggestion
            extra_metadata["transformer_repo"] = str(transformer_info.get("repository", {}).get("repUrl", ""))
```

Reply:
@julianpistorius This doesn't work for: `transformer_info = {"repository":None}`
<img width="403" alt="Screen Shot 2020-08-28 at 12 30 01 PM" src="https://user-images.githubusercontent.com/45463434/91608346-3a0c3c00-e92a-11ea-95f3-80fc477f1ac0.png">

Reply:
Oh, yeah, sorry. My suggestion will only work if the key `"repository"` doesn't exist at all in the `transformer_info` dictionary, but will fail if it exists and is set explicitly to None.

This should work, but now it's getting kinda ugly and less readable than your original:

```python
str((transformer_info.get("repository", {}) or {}).get("repUrl", ""))
```